tests/controllers/TalksControllerTest.php

```diff
@@ -644,7 +644,8 @@ public function testApproveClaimAsHostSucceeds()
         $request->user_id = 2;
         $request->parameters = [
             'username'      => 'psherman',
-            'display_name'  => 'P Sherman'
+            'display_name'  => 'P Sherman',
+            'action'        => 'approve',
```

Comment:
Not really needed the test works without this line too but this way is more correct